enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -92,41 +72,65 @@ scrape_configs:
         regex: container_(network_tcp_usage_total|tasks_state|cpu_usage_seconds_total|memory_failures_total|network_udp_usage_total)
         action: drop
 
-  # Scrape config for service endpoints.
+  # pushgateway
+
+  - job_name: 'pushgateway-exporter'
+    honor_labels: true
+
+    kubernetes_sd_configs:
+      - role: service
+
+    relabel_configs:
+      - source_labels: [__meta_kubernetes_service_annotation_{{ dot2underscore .PrometheusDomain }}_probe]
+        action: keep
+        regex: pushgateway
+
+  # Scrape config for probing services via the Blackbox Exporter.
   #
   # The relabeling allows the actual service scrape endpoint to be configured
   # via the following annotations:
   #
-  # * `{{ .PrometheusDomain }}/scrape`: Only scrape services that have a value of `true`
-  # * `{{ .PrometheusDomain }}/scheme`: If the metrics endpoint is secured then you will need
-  # to set this to `https` & most likely set the `tls_config` of the scrape config.
-  # * `{{ .PrometheusDomain }}/path`: If the metrics path is not `/metrics` override this.
-  # * `{{ .PrometheusDomain }}/port`: If the metrics are exposed on a different port to the
-  # service then set this appropriately.
-  - job_name: 'kubernetes-service-endpoints'
+  # * `{{ .PrometheusDomain }}/probe`: Only probe services that have a value of `true`
+
+  - job_name: 'prometheus-blackbox'
```

Comment:
Do we have a test for this probe stuff?

Reply:
no, this is just default prometheus config.  we don't use it